R/checks.R

```diff
@@ -178,3 +194,28 @@ CheckPredictionVariables <- function(object, newdata)
     }
     return(newdata)
 }
+
+checkPredictionWarningMessage <- function(label, levels, counts, outliers.removed = FALSE)
```

Comment:
createPredictionWarningMessage seems like a more appropriate name to me

Reply:
the parameter named `counts` could be made more descriptive. `na.cases.removed`?

Reply:
the parameter `outliers.removed` does not have the same meaning as in the outer function, so it'd be nice to rename it to more clearly describe what it actually represents: that the outlier removal is responsible for the level being removed from the training data and not just that outlier removal was performed.

Reply:
or you could just document the function :)

Reply:
Done, will do all above comments.